Raywire/iReporter
# Adds progressive image loading

**What does this PR do?**
Adds progressive image loading
**Descriptions of the tasks to be completed?**

- Add progressive image loading and animations on an incident to create the impression of continuity
- Add global functions to eslint settings

**How should this be manually tested?**
Go to https://raywire.github.io/iReporter/UI/redflags.html 
Navigate to an incident and as it loads you will notice the animations
